hack/vendor.sh

```diff
@@ -36,6 +36,11 @@ clone git github.com/hashicorp/consul v0.5.2
 # get distribution packages
 clone git github.com/docker/distribution 419bbc2da637d9b2a812be78ef8436df7caac70d
 
+clone git github.com/docker/notary 17af21f00c9fa4dc960ca8244f230ca082bd8745
+clone git github.com/endophage/gotuf ab4ba80203ffa5bfd742e6891bd28bfbf43a9453
+clone git github.com/tent/canonical-json-go 96e4ba3a7613a1216cbd1badca4efe382adea337
+clone git github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
```

Comment:
We are currently not signing using ed25519 but may need to verify these signatures.